react/features/authentication/actions.native.ts

```diff
@@ -65,11 +68,42 @@ export function cancelWaitForOwner() {
     };
 }
 
-/** .
+/**
  * Redirect to the default location (e.g. Welcome page).
  *
  * @returns {Function}
  */
 export function redirectToDefaultLocation() {
     return (dispatch: IStore['dispatch']) => dispatch(appNavigate(undefined));
 }
+
+/**
+ * Opens token auth URL page.
+ *
+ * @param {string} tokenAuthServiceUrl - Authentication service URL.
+ *
+ * @returns {Function}
+ */
+export function openTokenAuthUrl(tokenAuthServiceUrl: string) {
+    let url = appendURLHashParam(tokenAuthServiceUrl, 'skipPrejoin', 'true');
+
+    // Append ios=true or android=true to the token URL.
+    url = appendURLHashParam(url, Platform.OS, 'true');
+
+    return () => {
+        Linking.openURL(url);
+    };
+}
+
+/**
+ * Login.
+ *
+ * @returns {{
+ *     type: LOGIN
+ * }}
+ */
+export function login() {
```

Comment:
Can we move to actions.any and drop the one in actions.web?

Reply:
Sure